openlibrary/templates/type/author/view.html

```diff
@@ -64,20 +64,29 @@
 
 $set_share_links(url=request.canonical_url, title=title, view_context=ctx)
 
+$if page.birth_date or page.death_date:
+    $ entityTypeSchema = "https://schema.org/Person"
+$elif page.entity_type:
+    $ entityTypeSchema = "https://schema.org/Organization"
+
 <div id="contentHead">
 
     $:macros.databarView(page)
 
-    <h1>$title</h1>
-    <h2 class="author collapse">
-    $if page.birth_date or page.death_date:
-        $page.birth_date - $page.death_date
-    $else:
-        $if page.date:
-            $page.date
-    </h2>
+    <div itemscope itemtype=$entityTypeSchema>
+        <h1 itemprop="name">$title</h1>
+        <h2 class="author collapse">
+        $if page.birth_date or page.death_date:
+            <span itemprop="birthDate">$page.birth_date</span> - <span itemprop="deathDate">$page.death_date</span>
+        $else:
+            $if page.date:
+                $page.date
+        </h2>
+    </div>
+
 </div>
-<div id="contentBody">
+
+<div id="contentBody" itemscope itemtype=$entityTypeSchema>
```

Comment:
This will define a second `Person`; once we move the `itemscope` to the `body`, this should no longer be needed :+1:

```suggestion
<div id="contentBody">
```